client/src/components/App.jsx

```diff
@@ -1,29 +1,36 @@
-import React from 'react';
+import React from "react";
+import axios from "axios";
```

Comment:
I'd be interested to see what (if any) performance impact their is from using axios here instead of .fetch.  I'd been trying to avoid additional dependencies on the front-end, since it means more code to ship to the browser.  On the other hand, fetch has to be transpiled for older browsers, so maybe it's not *that* much more code to use axios.  

Any any event, it doesn't matter at this stage, but it could be something to circle back to when we turn to increasing page load time.

Reply:
I don't think it will increase performance, especially in a small app like ours but it is easier to use because it parses the api response for you so it does cut out a step from the api call. Also I think I read somewhere it is better at handling errors.

Reply:
Makes sense!  Thanks.